tsconfig.strictNullChecks.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Does the build not work if this is only present at the root?

Reply:
I see from your commit history that you tried that. :)

Reply:
It needs to be in both to satisfy both requirements. If the file is only present in the root then the build fails because it looks for it under packages\teams-js.